seqr/views/utils/file_utils.py

```diff
@@ -64,7 +68,9 @@ def parse_file(filename, stream):
 
 def _parse_excel_string_cell(cell):
     cell_value = cell.value
-    if cell_value and cell.data_type == 'n' and int(cell_value) == cell_value:
+    if not cell_value:
+        cell_value = ''
```

Comment:
This if/else needs to be in the other order, otherwise the value `0` will get removed when in reality we want it to be the string `"0"`

Reply:
Oh... Should I check `0` and `None` separately?

Reply:
No, like I said, change the order of the if/else:
```
if cell.data_type == 'n' and int(cell_value) == cell_value:
        cell_value = '{:.0f}'.format(cell_value)
elif not cell_value:
        cell_value = ''
```

Reply:
So, the number `0` will become a string `'0'`?